device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/transport/https/HttpsConnection.java

```diff
@@ -70,10 +70,10 @@ public HttpsConnection(URL url, HttpsMethod method, final ProxySettings proxySet
     {
         // Codes_SRS_HTTPSCONNECTION_11_022: [If the URI given does not use the HTTPS or HTTP protocol, the constructor shall throw an IllegalArgumentException.]
         final String protocol = url.getProtocol();
-        if (!protocol.equalsIgnoreCase("HTTPS") && !protocol.equalsIgnoreCase("HTTP"))
+        if (!protocol.equalsIgnoreCase("HTTPS"))
         {
```

Comment:
If we make this change we need to let PM know before drop it off. lets chat about this

Reply:
The method signature doesn't change and if someone uses http instead of https, the SSL context will throw an exception. so I don't think there is a behavior change. The exception we throw will be slightly different though.

Reply:
I took a look through the code, and there are no instances where the SDK would ever pass "http" instead of "https", so there is no impact on our customers here. We aren't taking away http support because we never had http support